src/locales/en/chapter4.js

```diff
@@ -133,7 +132,7 @@ export const lorraine = {
     "Do you know about the dragon?", // ann
     "Of course.", // architect
     "The dragon was sleeping in the basement of this temple, but woke up about half a month ago.", // architect
-    "Since then the prince has been visiting the dragon frequently.", // architect
+    "Since then the prince has been visiting the dragon frequently.", // architect // TODO: ask later
```

Comment:
I think the story has mentioned it before.  If you think it's confusing then you could make it "Since then, Prince Ethelbald has been visiting the dragon frequently."

Reply:
Thank you. so I added his name, because some town's events can be ingnored.